examples/ecommerce/src/App.mobile.css

```diff
@@ -0,0 +1,261 @@
+@media (max-width: 899px) {
```

Comment:
this can be done with a media query on the `link` too I think

Reply:
You're saying something like this, right?
```
  <link rel="stylesheet" type="text/css" href="App.mobile.css" media="max-width: 899px">
```

I don't know how to do that from `import './App.mobile.css';`. This should be easier on vanilla where they directly use `link` tag.